agent/api/statechange.go

```diff
@@ -89,19 +88,30 @@ type AttachmentStateChange struct {
 	Attachment *apieni.ENIAttachment
 }
 
+// SkippableStateChange is an error used to indicate that this state change can be omitted
+type SkippableStateChange struct {
```

Comment:
is this needed? seems like NewTaskStateChangeEvent and NewContainerStateChangeEvent only return this type of error, so we can just use the default error type

Reply:
sure I could leave default error type, I thought there was one instance where it _could_ return another error but now I cant find it 🤔